bucket_map/src/bucket_storage.rs

```diff
@@ -332,7 +332,9 @@ impl BucketStorage {
         self.stats.resizes.fetch_add(1, Ordering::Relaxed);
         self.stats.resize_us.fetch_add(m.as_us(), Ordering::Relaxed);
     }
-    pub fn num_cells(&self) -> u64 {
+
+    /// Return the number of cells currently allocated
+    pub fn capacity(&self) -> u64 {
```

Comment:
I think I was the one that changed this from capacity() to num_cells() initially, so it's only fair that I fix it because I was wrong 😅